### DISCUSSION TITLE
MQTT server example

### DISCUSSION THREAD

**Discussion by venkatou** [AxisCommunications/acap3-examples#189] [Q&A] [answered]
Hello, are there examples for sending event via mqtt using acap? Thanks!

**Comment by pataxis** [comment, marked as answer]
Hi @venkatou , thanks for your question, there is no MQTT example today.
But you could build an open source version of MQTT as a library and use in your application. Search the issue tracker for some previous suggestions of using MQTT and see the [utility-libraries](https://github.com/AxisCommunications/acap3-examples/tree/main/utility-libraries) for examples how to build an external library.
